src/colmap/controllers/incremental_mapper.h

```diff
@@ -114,6 +114,24 @@ struct IncrementalMapperOptions {
   int ba_global_max_refinements = 5;
   double ba_global_max_refinement_change = 0.0005;
 
+  // Whether to use prior camera positions
+  bool use_prior_position = false;
+
+  // Whether to setup the prior positions from database
+  bool set_prior_position_from_database = true;
+
+  // Standard deviation on the position priors
+  double prior_position_std_x = 1.;
+  double prior_position_std_y = 1.;
+  double prior_position_std_z = 1.;
```

Comment:
Would it make sense to extend `PosePrior` with a full covariance such has each measurement can have a different noise?

Reply:
Good idea.  I will take care of it.

Reply:
Yes, +1

Reply:
:+1:

Reply:
Thank you!